libhif/hif-types.h

```diff
@@ -82,6 +83,7 @@ typedef enum {
         HIF_ERROR_REPO_NOT_FOUND                = 19,   /* Since: 0.1.0 */
         HIF_ERROR_FAILED_CONFIG_PARSING         = 24,   /* Since: 0.1.0 */
         HIF_ERROR_PACKAGE_NOT_FOUND             = 8,    /* Since: 0.1.0 */
+        HIF_ERROR_NO_SPACE                      = 46,   /* Since: 0.2.3 */
```

Comment:
Yup, I want it in F23 and F24 for the distro upgrades work, definitely need it in 0.2.3.